Sources/XcodesKit/Version+.swift

```diff
@@ -1,47 +1,24 @@
 import Version
 
 public extension Version {
-    func isEqualWithoutBuildMetadataIdentifiers(to other: Version) -> Bool {
-        return major == other.major && 
-               minor == other.minor &&
-               patch == other.patch &&
-               prereleaseIdentifiers == other.prereleaseIdentifiers
-    }
-
-    func isEqualWithoutAllIdentifiers(to other: Version) -> Bool {
-        return major == other.major &&
-               minor == other.minor &&
-               patch == other.patch
-    }
-
-    /// If release versions, don't compare build metadata because that's not provided in the /downloads/more list
-    /// if beta versions, compare build metadata because it's available in versions.plist
-    func isEquivalentForDeterminingIfInstalled(toInstalled installed: Version) -> Bool {
-        let isBeta = !prereleaseIdentifiers.isEmpty
-        let otherIsBeta = !installed.prereleaseIdentifiers.isEmpty
-
-        if isBeta && otherIsBeta {
-            if buildMetadataIdentifiers.isEmpty {
-                return major == installed.major &&
-                       minor == installed.minor &&
-                       patch == installed.patch &&
-                       prereleaseIdentifiers == installed.prereleaseIdentifiers
-            }
-            else {
-                return major == installed.major &&
-                       minor == installed.minor &&
-                       patch == installed.patch &&
-                       prereleaseIdentifiers == installed.prereleaseIdentifiers &&
-                       buildMetadataIdentifiers.map { $0.lowercased() } == installed.buildMetadataIdentifiers.map { $0.lowercased() }
-            }
-        }
-        else if !isBeta && !otherIsBeta {
-            return major == installed.major && 
-                   minor == installed.minor &&
-                   patch == installed.patch
+    /// Determines if two Xcode versions should be treated equivalently. This is not the same as equality.
+    /// 
+    /// We need a way to determine if two Xcode versions are the same without always having full information, and supporting different data sources.
+    /// For example, the Apple data source often doesn't have build metadata identifiers.  
+    func isEquivalent(to other: Version) -> Bool {
+        // If we don't have build metadata identifiers for both Versions, compare major, minor, patch and prerelease identifiers.
+        if buildMetadataIdentifiers.isEmpty || other.buildMetadataIdentifiers.isEmpty {
+            return major == other.major &&
+                   minor == other.minor &&
+                   patch == other.patch &&
+                   prereleaseIdentifiers.map { $0.lowercased() } == other.prereleaseIdentifiers.map { $0.lowercased() }
```

Comment:
The first 3 checks have been pulled up into `isEqualWithoutAllIdentifiers(to other: Version) -> Bool`. Could that be incorporated here to avoid repeating?
```suggestion
            return isEqualWithoutAllIdentifiers(to: other) &&
                   prereleaseIdentifiers.map { $0.lowercased() } == other.prereleaseIdentifiers.map { $0.lowercased() }
```